lib/ansible/modules/cloud/azure/azure_rm_policysetdefinition.py

```diff
@@ -0,0 +1,370 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Fan Qiu, (fanqiu@microsoft.com)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_policysetdefinition
```

Comment:
the name is policysetdefinition, not policydefinition? is it from sdk model name?

Reply:
There are three modules about policy, which are policy definition, policy assignment and policy set definition. There is another PR about policy definition.